README.md

```diff
@@ -38,11 +38,19 @@ Options are configured via environment variables.
 | LOGGER_SECRETS    | No       | ["secret","secret"]                        | String array of exact secrets to sanitize in logs                                                                            |
 | DRY_RUN           | No       | false                                      | Run the service without actually sending out exit messages                                                                   |
 
+## Preparing Exit Messages
+
+Once you generate and sign exit messages, you can encrypt them for storage safety.
+
+Exit messages are encrypted and decrypted following [EIP-2335](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2335.md) spec.
+
+You can check a simple example in TS in `encrypt.ts`.
+
 ## Running
 
 Either:
 
-- Use Docker image from Docker Hub: TODO
+- Use a Docker image from [Docker Hub](https://hub.docker.com/r/lidofinance/validator-ejector)
```

Comment:
I would propose to fix the hash of the dockerimage in docs, like
`lidofinance/validator-ejector@sha256:8a68fac91d29cebecd198c14f73d7b50f91e3a5c3c9e99015a52ab644a6b2257`

Reply:
I agree, but for release versions. Right now we only have `dev` versions for testing.